src/main/java/edu/cornell/kfs/sec/document/CuSecurityModelMaintainableImpl.java

```diff
@@ -0,0 +1,12 @@
+package edu.cornell.kfs.sec.document;
+
+import org.kuali.kfs.sec.businessobject.SecurityModel;
+import org.kuali.kfs.sec.document.SecurityModelMaintainableImpl;
+
+public class CuSecurityModelMaintainableImpl extends SecurityModelMaintainableImpl {
+
+    protected String buildModelRoleId(SecurityModel securityModel) {
```

Comment:
Please add the "Override" annotation if this overriding a superclass method.

Reply:
Good point! Adding